project/apps/blog/src/main.ts

```diff
@@ -3,19 +3,35 @@
  * This is only a minimal backend to get started.
  */
 
-import { Logger } from '@nestjs/common';
+import { Logger, ValidationPipe } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app/app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { ConfigService } from '@nestjs/config';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const globalPrefix = 'api';
+  const config = new DocumentBuilder()
+    .setTitle('Blog service')
+    .setDescription('Blog service API')
+    .setVersion('1.0')
+    .build();
+
   app.setGlobalPrefix(globalPrefix);
-  const port = process.env.PORT || 3000;
+
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('spec', app, document);
+
+  app.useGlobalPipes(new ValidationPipe({
+    transform: true
+  }));
+
+  const configService = await app.resolve(ConfigService);
+  const port = configService.get('application.port') || 3000;
```

Comment:
Нужно вынести числа в константах. Это одно критерий
Так же : [Б10.](https://up.htmlacademy.ru/nodejs-2/9/criteries#b10) Список констант идёт перед основным кодом.

```suggestion
  const port = configService.get('application.port') || DEFAULT_PORT;
```